app/services/find/cycle_timetable.rb

```diff
@@ -39,17 +39,15 @@ class CycleTimetable
         find_opens: Time.zone.local(2024, 10, 1, 9), # CONFIRMED
         apply_opens: Time.zone.local(2024, 10, 8, 9), # CONFIRMED
         first_deadline_banner: Time.zone.local(2025, 7, 12, 9), # TBC
-        apply_deadline: Time.zone.local(2025, 9, 17, 18), # CONFIRMED
-        find_closes: Time.zone.local(2025, 9, 25, 23, 59, 59) # TBC
+        apply_deadline: Time.zone.local(2025, 9, 16, 18), # CONFIRMED
+        find_closes: Time.zone.local(2025, 9, 30, 23, 59, 59) # TBC
```

Comment:
Is the `find_closes` date for the 2025 cycle still not confirmed? 

Are the dates below for the 2026 cycle confirmed?

Reply:
I'm being quite fussy because I know we don't usually look at these dates for a year after Find opens.

Reply:
We are publishing the 2025-26 dates on the gov uk pages and they have been ok-ed by Eliot. I suppose there is a possibility that the find_closes date will change based on the 20-27 find_open date, but that is a problem for 2027 us. https://ukgovernmentdfe.slack.com/archives/C077NRS8K8D/p1723540691902549